src/color_laws.jl

```diff
@@ -370,3 +370,63 @@ function f99_invum(x::Real, Rv::Real)
             optnir_axebv_y,
         )
 end
+
+# read and unpack tabulated data
+const data_x, data_k, data_delta_k, data_sigma_k = let data = readdlm(joinpath(dirname(pathof(DustExtinction)), "data", "F19_tabulated.dat"), skipstart=1)
```

Comment:
Sorry, think my question was lost in the merge. Should we be worried that AppVeyor didn't seem to catch the forward slash I accidentally left behind?

Reply:
No, but I'm surprised it didn't complain.